soh/src/overlays/actors/ovl_En_Ge2/z_en_ge2.c

```diff
@@ -54,6 +54,7 @@ void EnGe2_Walk(EnGe2* this, PlayState* play);
 void EnGe2_Stand(EnGe2* this, PlayState* play);
 void EnGe2_WaitLookAtPlayer(EnGe2* this, PlayState* play);
 void EnGe2_ForceTalk(EnGe2* this, PlayState* play);
+void EnGe2_SetupCapturePlayer(EnGe2* this, PlayState* play);
```

Comment:
this doesn't need to be added to this file does it? i'm assuming this is leftover from building this out before moving things to use a VB_SHOULD?

Reply:
No, but it seems like a miss not listing prototype

Reply:
removed